nav2_bringup/bringup/launch/tb3_simulation_launch.py

```diff
@@ -141,6 +165,8 @@ def generate_launch_description():
         cwd=[launch_dir], output='screen')
 
     urdf = os.path.join(bringup_dir, 'urdf', 'turtlebot3_waffle.urdf')
+    with open(urdf, 'r') as infp:
```

Comment:
Is this necessary? It looks before like we just gave `urdf` as an argument to the robot state publisher and it just handled it itself? Just asking the question

Reply:
there is a warning if we gave urdf as an argument.
[WARN 1626405835.205918208] [robot_state_publisher]: No robot_description parameter, but command-line argument available.  Assuming argument is name of URDF file.  This backwards compatibility fallback will be removed in the future. (RobotStatePublisher() at /tmp/binarydeb/ros-galactic-robot-state-publisher-2.4.3/src/robot_state_publisher.cpp:85)  

to remove this the urdf should be converted to robot description parameter

Reply:
Got it, thanks for fixing that